accounts-db/src/ancient_append_vecs.rs

```diff
@@ -2730,6 +2751,40 @@ pub mod tests {
         PackedStorages,
     }
 
+    pub fn build_accounts_from_storage_with_slot(
```

Comment:
lots of mess getting things that are of the form `&([(&a, &b)])` or similar given various `Vec`s.